platform/node/cmake/module.cmake

```diff
@@ -160,7 +160,7 @@ function(add_node_module NAME)
             )
         endif()
 
-        target_link_libraries(${_TARGET} PRIVATE ${NAME})
+        target_link_libraries(${_TARGET} PRIVATE ${NAME} mbgl-compiler-options)
```

Comment:
What is the benefit of passing this on?

Reply:
The main benefit is the consistency in compile flags being shared with all targets. This ensures, for example, the code requirements are the same in all source files. Without this option enabled, some things may pass in one target but not in another, like C4244 warnings (in current compille options it should not pass).

Reply:
`mbgl-compiler-options` target is defined here:

https://github.com/maplibre/maplibre-gl-native/blob/1341f33ba425b4115028a8e54d337c432eb46a33/CMakeLists.txt#L38

Reply:
Thanks